.travis.yml

```diff
@@ -3,6 +3,8 @@ dist: focal
 language: python
 python: '3.8'
 
+if: (branch = main OR tag IS present) AND (type = push)
```

Comment:
This should also allow to easily change this to `type = pull_request` when working on a PR that affects Travis

Reply:
What does `tag IS present` mean?

Reply:
My understanding is that it will then also run on tagged commits. 

It's not fully clear to me whether it is important to include this, as the commit we tag for a release will typically also be on the main branch (and if we at some point have a release branch, we should include that branch here). Although based on https://docs.travis-ci.com/user/customizing-the-build/#safelisting-or-blocklisting-branches, it might be needed.